scripts/tooltip.js

```diff
@@ -10,7 +10,8 @@ function initToolTipText() {
 		makeFirstSample: "Create your first ever sample! Requires " + game.beatsPerSample + " beats.",
 		makeFirstSong: "Active Task: Make your first ever song! Requires " + game.samplesPerSong + " samples and takes 10 seconds to complete.",
 		makeNewSong: "Active Task: Make a new song! Requires " + game.samplesPerSong + " samples and takes 10 seconds to complete.",
-		djBirthdayParty: "Active Task: DJ for a birthday party. Rewards $50 and 250 Laptop XP. Completes over 120 seconds.",
-		buyNewLaptop: "Purchase a new laptop. Costs $500. Reduces the number of clicks per beat to 5."
+		djBirthdayParty: "Active Task: DJ for a birthday party. Requires 30 beats, rewards $50 and 250 Laptop XP and completes over 120 seconds.",
```

Comment:
I added the # of beats requirement to the tooltip and removed it from the output message when you don't have enough beats